### PR TITLE
style: move chip styling to theme

### PR DESCRIPTION
**Pull request by shawnyama**
# Description

Move chip styling to theme to keep things consistent

### CODE REVIEW

File: packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss [hunk 1]
```diff
@@ -1,15 +1,24 @@
 @use 'sass:math';
 
 .p-chip {
-    background-color: $chipBg;
-    color: $chipTextColor;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss:5]: Could you please leave those SCSS variables in place, and edit their definition in [the `_variable.scss` file](https://github.com/DARPA-ASKEM/TERArium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L781-L782).

**Review by YohannParis — APPROVED** [review]

**Comment by shawnyama** [comment]
Not sure how but my local accordion-theme branch ended up merging with this one (just a few more adjustments)

File: packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss [hunk 2]
```diff
@@ -4,12 +4,21 @@
     background-color: $chipBg;
     color: $chipTextColor;
     border-radius: $chipBorderRadius;
-    padding: 0 nth($inputPadding, 2);
+    font-weight: bold;
+    font-size: var(--font-caption);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss:8]:
```suggestion
    font-weight: $fontWeightSemiBold;
    font-size: $fontCaption;
```

File: packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss [hunk 3]
```diff
@@ -24,10 +33,11 @@
     }
 
     .p-chip-remove-icon {
+        font-size: var(--font-caption);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/designer/components/misc/_chip.scss:36]:
```suggestion
        font-size: $fontCaption;
```

**Review by YohannParis — APPROVED** [review]

**Review by YohannParis — APPROVED** [review]

**Pull request merged**